### ISSUE TITLE
Init script

### ISSUE THREAD

**Issue by reavessm** [reavessm/MDS#3]
Add initialization script to inheritance framework.

```bash
read -p "Would you like add a container? " ans
while [[ $ans == [yY] ]]
do
    new
    read -p "Would you like to add another? " ans
done

# Subshell used to handle relative paths and return to cur dir
(cd proxy.d && autogen.sh)
```

The read statements should be replaced with `dialog` for coherency.

**Comment by reavessm** [comment]
We should also probably open the new container in $EDITOR to allow creating of 'exposedPort' or else the autoconfig is pretty pointless

**Comment by reavessm** [comment]
Fixed with 20b89cdd0c52d1a8d3cc8d3a31777262e81ad9d8
